scipy/stats/_stats_py.py

```diff
@@ -889,11 +894,11 @@ def tsem(a, limits=None, inclusive=(True, True), axis=0, ddof=1):
 
 
 def _moment_outputs(kwds):
-    moment = np.atleast_1d(kwds.get('order', 1))
-    if moment.size == 0:
+    order = np.atleast_1d(kwds.get('order', 1))
```

Comment:
I probably shouldn't have complicated this PR by renaming the old variable `moment` to `order`... but I did.